### PR TITLE
[DCOS-46650] Cassandra base tech upgrade to 3.11.3

### PR DESCRIPTION
**Pull request by soumasish**
## What does this PR aim to do?
- Upgrade Cassandra base tech version for cassandra package release.
- Defaults to the package CQL sh version which is 5.0.1

## How were the changes tested?
- By running a CI build.

### CODE REVIEW

**Comment by samvantran** [comment]
Looks like there might be some issues with the base tech upgrade from CI test failures

**Comment by soumasish** [comment]
@akirillov can you take a look please.

**Comment by soumasish** [comment]
@samvantran @akirillov one more look please.

File: frameworks/cassandra/tests/config.py [hunk 1]
```diff
@@ -84,7 +80,7 @@ def _get_test_job(
         "id": "test.cassandra." + name,
         "run": {
             "cmd": " && ".join(commands),
-            "docker": {"image": "cassandra:3.0.13"},
+            "docker": {"image": "cassandra:3.11.3"},
```
Comment (akirillov) [inline frameworks/cassandra/tests/config.py]: let's introduce a variable for Cassandra version

Comment (soumasish) [reply frameworks/cassandra/tests/config.py]: Ok will update that.

File: frameworks/cassandra/universe/resource.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
       "cassandra-openssl-tar-gz": "https://downloads.mesosphere.com/cassandra/assets/openssl-1.0.2n-libs.tar.gz",
       "libmesos-bundle-tar-gz": "{{libmesos-bundle-url}}",
       "scheduler-zip": "{{artifact-dir}}/cassandra-scheduler.zip",
-      "cassandra-tar-gz": "https://downloads.mesosphere.com/cassandra/assets/apache-cassandra-3.0.16-bin-dcos.tar.gz",
+      "cassandra-tar-gz": "https://downloads.mesosphere.com/cassandra/assets/apache-cassandra-3.11.3-bin.tar.gz",
```
Comment (samvantran) [inline frameworks/cassandra/universe/resource.json]: is this straight from apache cassandra downloads or did you have to modify the tarball with dcos bits? I ask bc all previous versions in the `assets/` folder includes the `-dcos` suffix. If so, pls rename and reupload

**Comment by soumasish** [comment]
@akirillov one last time.

**Comment by akirillov** [comment]
Thanks, @soumasish, LGTM. Let's wait for CI builds to finish.

**Pull request merged**